File: features/digital/flight/src/main/java/com/tokopedia/flight/orderlist/data/cloud/entity/OrderEntity.java
package com.tokopedia.flight.orderlist.data.cloud.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * @author by alvarisi on 12/6/17.
 */

public class OrderEntity {
    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("attributes")
    @Expose
    private AttributeEntity attributes;

    public String getType() {
        return type;
    }

    public String getId() {
        return id;
    }

    public AttributeEntity getAttributes() {
        return attributes;
    }
}

File: features/travel/flight_orderlist/src/main/java/com/tokopedia/flight/orderlist/data/cloud/entity/OrderEntity.kt
package com.tokopedia.flight.orderlist.data.cloud.entity

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class OrderEntity(
        @SerializedName("type")
        @Expose
        val type: String = "",
        @SerializedName("id")
        @Expose
        val id: String = "",
        @SerializedName("attributes")
        @Expose
        val attributes: AttributeEntity)
